user/pages/05.Background/02.local-storage/docs.en.md

```diff
@@ -66,11 +66,9 @@ Local SSD storage instances cannot be (live) migrated, we need to regularly rebo
 
 To keep the impact predictable, we hereby announce a regular maintenance window:
 
->Every week in the night from wednesday, 11pm (23h) to thursday, 4am,
-
-we will restart about 25% of our local ssd storage compute nodes. Statistically, every instance
-
-> will be shut down once a month.
```

Comment:
those lines (67,73) were intentionally highlighted to stand out against the rest of the text. I think you proposal throws them together with each other and line 71.

Reply:
But they look light gray (against a white background), which is less visible than normal text. The middle line isn't even the same as the first and last line. 
<img width="699" alt="image" src="https://user-images.githubusercontent.com/36202147/110447733-cd37e280-80c0-11eb-8c13-b5d48f6660bb.png">

The new version is red.
<img width="675" alt="image" src="https://user-images.githubusercontent.com/36202147/110447804-dfb21c00-80c0-11eb-8440-80a660ee0af6.png">

Reply:
I agree, that red is better than grey. The quotation was intended rather as a logical markup than a color choice. I propose to highlight the recurring date (1st line) with a bangbang / double exclamation mark and perhaps moving "maintenance window:" into that box. The rest should imo remain normal text with the formerly quoted words highlighted as cursive, italics, slanted or bold.

Reply:
Yes, this looks ok to me:
```
### When is the local SSD storage node maintenance window?

To keep the impact predictable, we hereby announce a regular

!! maintenance window:
!! Every week in the night from wednesday, 11pm (23h) to thursday, 4am,

**we will restart about 25% of our local ssd storage compute nodes.
Statistically, every instance will be shut down once a month.**

```
![image](https://user-images.githubusercontent.com/36202147/110651882-233d8080-81bc-11eb-883c-15793de80606.png)